generator/file_test.go

```diff
@@ -124,6 +125,52 @@ var _ = Describe("File", func() {
 		)
 	})
 
+	Describe("ProcessFile", func() {
+		Context("when get a header", func() {
+			var f *descriptor.FileDescriptorProto
+
+			BeforeEach(func() {
+				f = &descriptor.FileDescriptorProto{
+					Options: &descriptor.FileOptions{},
+					Name:    sp("product.proto"),
+					Package: sp("pb"),
+					MessageType: []*descriptor.DescriptorProto{
+						{
+							Name: sp("Product"),
+							Field: []*descriptor.FieldDescriptorProto{
+								&descriptor.FieldDescriptorProto{
+									Name:     sp("ID"),
```

Comment:
Could you, please:
* Rebase master onto your branch
* Rename this field to `id` (lower case). For details see #14 

Thanks!

Reply:
Thanks for the feedback. I've addressed the above.